hashicorp/dev-portal
# 🛹 feat: adds bookmark toast

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksadds-bookmark-toast-hashicorp.vercel.app/nomad/tutorials) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1202791375540729) 🎟️

## 🗒️ What

This PR adds a toast notification for the bookmark handler. 

## 📸 Design Screenshots

<img width="1129" alt="Screen Shot 2022-08-17 at 12 17 08 PM" src="https://user-images.githubusercontent.com/36613477/185224253-979e670e-249e-4391-a3b8-2965059f981f.png">

## 🧪 Testing

- Visit the[ swingset page](https://dev-portal-git-ksadds-bookmark-toast-hashicorp.vercel.app/swingset/components/tutorialcard)
- Click the bookmark icon to see the toast

**_UPDATE_** I removed the integration from this PR, so the test steps listed above won't work. It will be re-added when we do the full data integration. This PR will just serve as a stage for the `makeBookmarkToast` function. 

## 💭 Anything else?

We will hook this toast up to the success of the data calls & render the tutorial name in follow up with. This is just a skateboard to set-up the UI / package the toast generation into a single function.
